relationships/synthesis/SYNTHETIC-MONITOR-to-APM-APPLICATION.yml

```diff
@@ -0,0 +1,22 @@
+relationships:
+  - name: syntheticMonitorCallsApmApplication
+    version: "1"
+    origins:
+      - <Need Transaction Events from APM agent as source>
```

Comment:
need this to be done: https://new-relic.atlassian.net/browse/NR-231098
to add the appropriate origin before I can merge